### PR TITLE
Visualize target object posterior image as an overlaid image

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 1]
```diff
@@ -2,8 +2,11 @@
   <arg name="DIST"/>
   <arg name="HEIGHT"/>
   <arg name="POSTERIOR"/>
+  <arg name="POSTERIOR_UNMASK"/>
   <arg name="INPUT_IMAGE"/>
   <arg name="MASK"/>
+  <arg name="MASKED_INPUT"/>
+  <arg name="CLASS_LABEL"/>
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: `Label_Image` is better?

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 2]
```diff
@@ -30,17 +38,35 @@
...
   </node>
   <node name="$(arg HAND)_sib_target_label_image_decomposer"
         pkg="jsk_perception" type="label_image_decomposer.py">
-    <remap from="~input" to="/$(arg HAND)_softkinetic_camera_resized/image_raw" />
+    <remap from="~input" to="$(arg INPUT_IMAGE)" />
     <remap from="~input/label" to="/$(arg HAND)_target_mask_to_label/output" />
   </node>
+
+  <node name="$(arg HAND)_sib_overlay_dist_on_color"
+    pkg="jsk_perception" type="overlay_image_color_on_mono">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: Indent.

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 3]
```diff
@@ -30,17 +38,35 @@
...
+    <remap from="~input/mono" to="$(arg INPUT_IMAGE)"/>
+    <remap from="~input/color" to="/$(arg HAND)_colorized_sib_dist/output"/>
+    <rosparam>
+      approximate_sync: true
+    </rosparam>
+  </node>
+
+  <node name="$(arg HAND)_sib_overlay_image_color_posteior_on_color"
+    pkg="jsk_perception" type="overlay_image_color_on_mono">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: Indent.

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 4]
```diff
@@ -30,17 +38,35 @@
...
+  <node name="$(arg HAND)_sib_overlay_dist_on_color"
+    pkg="jsk_perception" type="overlay_image_color_on_mono">
+    <remap from="~input/mono" to="$(arg INPUT_IMAGE)"/>
+    <remap from="~input/color" to="/$(arg HAND)_colorized_sib_dist/output"/>
+    <rosparam>
+      approximate_sync: true
+    </rosparam>
+  </node>
+
+  <node name="$(arg HAND)_sib_overlay_image_color_posteior_on_color"
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: Typo.

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 5]
```diff
@@ -30,17 +38,35 @@
...
+    pkg="jsk_perception" type="overlay_image_color_on_mono">
+    <remap from="~input/mono" to="$(arg INPUT_IMAGE)"/>
+    <remap from="~input/color" to="/$(arg HAND)_colorized_sib_posterior_unmask/output"/>
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: Typo.

**Comment by yuyu2172** [comment]
fixed

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 6]
```diff
@@ -2,45 +2,71 @@
...
+  <arg name="POSTERIOR_UNMASK"/>
   <arg name="INPUT_IMAGE"/>
   <arg name="MASK"/>
+  <arg name="MASKED_INPUT"/>
+  <arg name="LABEL_IMAGE"/>
   <arg name="HAND"/>
 
   <node pkg="jsk_perception" type="colorize_float_image"
-      name="$(arg HAND)_colorized_sib_posterior">
+        name="$(arg HAND)_colorized_sib_posterior">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: If you change this, it is better to be
```
<node name="..."
           pkg="..." type="...">
  ...
</node>
```

File: jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch [hunk 7]
```diff
@@ -2,45 +2,71 @@
...
   </node>
 
   <node pkg="jsk_perception" type="colorize_float_image"
-      name="$(arg HAND)_colorized_sib_height">
+        name="$(arg HAND)_colorized_sib_posterior_unmask">
+  <remap from="~input" to="$(arg POSTERIOR_UNMASK)"/>
+  </node>
+
+  <node pkg="jsk_perception" type="colorize_float_image"
+        name="$(arg HAND)_colorized_sib_height">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]:
```
 <node name="$(arg HAND)_colorized_sib_height"
       pkg="jsk_perception" type="colorize_float_image">
```

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: thanks.
I should add a vim plugin that makes templates for me.

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: In my vim environment, there is snippet for `<node>`

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/launch/include/sib_visualization.launch]: I appreciate it.

**Comment by wkentaro** [comment]
+1

**Pull request merged**